### DISCUSSION TITLE
Ergebnisse

### DISCUSSION THREAD

**Discussion by SsaraBab** [SsaraBab/Gruppenarbeit-WA#39] [General]
Ihr könnt anfangen zu diskutieren, sobald ihr bereit seid!

**Comment by ibrahim-tudo** [comment]
(d) Mathe und Mathe Leistungskurs haben eine negative lineare Zusammenhang während der zwischen Programmierung und Mathe-Leistungskurs positiv ist. Es gibt mehere Studenten die Programmierung studieren aber sich nicht am Leistungskurs interessieren. Mathe Studierenden sind abber fast gleich auf 'interessiert' und 'nicht interessiert aufgeteilt'

**Reply by MawKi2** [reply]
Hier sind die Ergebnisse aus der Anwendung von a und b:
Die deskriptive Analyse der metrischen Variablen ergab ein Durchschnittsalter von 25,3 Jahren (im Median 25) der Befragten, einen Mittelwert von 4,3 beim Interesse in Mathe und einen Mittelwert von 4 beim Interesse in Programmierung. Die Standardabweichung beträgt in allen Fällen ca. 2,1.
Bei den kategorialen zeigt sich eine Häufung bei den 23-27 jährigen (insg. 0,8 rel. Häufigkeit) beim Alter. Beim Mathe Interesse gibt es ein große Häufigkeit der 7, alle anderen Wertesind sehr gleich verteilt. Beim Interesse Programmierung sind 1, 2 und 7 häufiger vertreten. Nach Fächern ist Mathe im Vergleich wesentlich schwächer vertreten (nur 0,07 rel. Häufigkeit). Die Aufteilung nach Mathe LK ist nahezu 50/50.

**Reply by SsaraBab** [reply]
Da das Alter normalverteilt um den Erwartungswert 25 mit einer Standardabweichung von 2 programmiert ist, scheint hier alles geklappt zu haben! Auch die anderen Werte stimmen hier so wie Renée und ich sie uns vorgestellt hatten

**Comment by ibrahim-tudo** [comment]
(c) In alle Fächern, interessieren sich mehrere Studierenden an dem Mathe-Leistungskurs. Nur Statistik ist eine Ausnahme

**Reply by MawKi2** [reply]
Die Verteilung für Mathe- und für Programmierinteresse im Bezug auf Personen die Mathe LK Ja hatten ist sehr ähnlich der Verteilung des Mathe- u. Programmierinteresses mit der besonderheit das bei Mathe die Ausprägung bei 7 noch stärker ist.

**Comment by Moshirimozafar** [comment]
Mit Hilfe der Funktion f wurde von uns eine geeignete Visualisierung von drei Kategoriellen Variablen erstellt und mit ggplot Library ein Balkendiagramm von Faechern, programmierung und Mathe dargestellt.
Die Visualisierung zeigt, dass beispielsweise die Personen ein weinges Interesse an Mthematik haben. Sowie sind Interesse an Programmierung zwischen den Personen ganz unterschiedlich (Es giben ein hohes, weniges und mittleres Interesse an Programmierung).

**Reply by reneesch** [reply]
Interessant. An dich, aber auch an alle anderen von Teil 4: könntet ihr die von R ausgegebenen Grafiken hier reinstellen? Man kann hier Dateien hinzufügen. Es würde mich sehr interessieren, das zu sehen.

**Reply by Moshirimozafar** [reply]
Ja klar, sehr gerne ;)

**Reply by Moshirimozafar** [reply]
![28 02 2022_22 57 50_REC](https://user-images.githubusercontent.com/93552347/156065483-dab5bb2e-6708-4cb4-bcdf-ec2c673e3fb8.png)

**Reply by reneesch** [reply]
Danke fürs Hochladen! Hast du vielleicht auch die Grafik von derselben Funktion, nur mit 4 kategoriellen Variablen?

**Reply by ibrahim-tudo** [reply]
![010a4e5d-811a-4161-98c8-0b5829b53a11](https://user-images.githubusercontent.com/97898593/156072075-e23f2d6f-e4ef-49d4-b42d-1900f5d135e9.png)

**Comment by SsaraBab** [comment]
Hat irgendwer noch Ergebnisse für e)? Oder sonstige Anmerkungen?

**Reply by ibrahim-tudo** [reply]
(Es geht um die Interesse an Programmierung)
Ja. Mit einem niedrigen Quantil = 1/3 und mitteleren = 2/3 sind "low" am häufigsten zu sehen (mehr als 40). "Medium" sind ungefähr 35 und "High" ungefähr 20 

![Screenshot from 2022-02-28 21-58-24](https://user-images.githubusercontent.com/97898593/156058255-73426ba7-38d6-4cab-9e44-bbe7207aec5e.png)


Mit einem niedrigen Quantil = 0 und mitteleren = 1/2 sind "high" und "medium" fast gleich (ungefähr 40-42) während "low" ironischerweise relative low sind (ungefahr 15)

![Screenshot from 2022-02-28 22-02-56](https://user-images.githubusercontent.com/97898593/156058266-5b1959f4-9c78-4bf3-b959-22569b995555.png)

**Reply by reneesch** [reply]
Danke, Ibrahim! 

Hast du vielleicht auch noch Grafiken von f)?

**Reply by ibrahim-tudo** [reply]
https://github.com/SsaraBab/Gruppenarbeit-WA/discussions/39#discussioncomment-2267219

**Comment by reneesch** [comment]
Ah, schade, dass meine Funktion grafik.metrisch.dichotom() aus Zeile 214 im Funktionen-R-Skript1 nicht angewendet wurde, obwohl ich woanders danach gefragt hatte. Naja, einen Versuch ist es wert: 

Falls einer von euch noch Zeit hätte, die Funktion eben auszuprobieren, würde ich mich sehr freuen. In der Gruppe hatte ja jemand gesagt, er wolle es ausprobieren. (Ich markiere hier bewusst niemanden ;-)

**Reply by ibrahim-tudo** [reply]
![7405dd40-fdbd-4b64-9df6-d275b894b0d8](https://user-images.githubusercontent.com/97898593/156063550-4b56dbbd-c32f-48c8-8e22-74179ba726e7.png)
![0c794450-b1f8-4110-8a19-c9ffb087bd73 (1)](https://user-images.githubusercontent.com/97898593/156063571-8b720462-f195-433c-9bc2-6e6a6809e289.png)

**Reply by reneesch** [reply]
Danke fürs Hochladen der Grafiken! Mal sehen, ob noch jemand anderes was zur Interpretation sagt, bevor Sara und ich die Zusammenhänge enthüllen.

**Comment by SsaraBab** [comment]
Als kleine Anmerkung: 
Da Renée und ich morgen relativ früh raus müssen, würden wir gerne so gegen viertel nach 11 die Zusammenhänge, die wir ursprünglich programmiert haben, hier nochmal auflisten. Also wäre es gut, wenn ihr eure Ergebnisse vorher postet, wenn ihr noch daran arbeitet, und dann kann später immernoch dazu kommentiert werden, falls nötig

**Reply by SsaraBab** [reply]
!Falls ihr noch Ergebnisse posten und für euch rausfinden wollt, noch nicht lesen!

Hier die Zusammenhänge für das Studienfach und der Mathe_LK:
Für das Studienfach haben wir eine einfache sample Funktion genommen und die folgendermaßen verteilt:
- Statistik und Data Science jeweils mit 0.3
- Mathe mit 0.1
- Informatik mit 0.2

Der Mathe_LK wurde ohne Zusammenhang einfach mit einem sample erstellt, also ist hier jeder Zusammenhang, der gefunden wurde, rein zufälig aufgetaucht.

**Reply by SsaraBab** [reply]
Den Zusammenhang des Alters habe ich weiter oben (erster Eintrag) schon einmal in einer Antwort formuliert, falls es jemand übersehen hat.

**Comment by reneesch** [comment]
!Falls ihr noch Ergebnisse posten und für euch rausfinden wollt, noch nicht lesen!



Zu **Interesse an Mathe**: Wir haben im ersten Schritt 100 Werte aus der diskreten Gleichverteilung von 1 bis 7 mit sample() gezogen. Dann für Personen mit Studienfach "Mathe" die gezogenen Werte um 2 erhöht, für Personen mit Studienfach "Statistik" oder "Data Science" jeweils um 1. 
Das führte zu Werten höher als 7. Dafür haben wir eine Funktion geschrieben, die alle Werte höher als 7 auf 7 reduziert. 

Zu **Interesse an Programmieren**: Erster Schritt der selbe wie oben, also 100 Werte aus der diskreten Gleichverteilung von 1 bis 7 gezogen. Nun für Personen mit Studienfach "Informatik" die Werte mit 1.5 multipliziert, für Personen mit Studienfach "Data Science" mit 1.2 multipliziert. 
Danach wieder wegen Werten höher als 7 dieselbe Funktion benutzt, um diese Werte auf 7 zu reduzieren.
 Im letzten Schritt dann die entstandenen Werte gerundet, damit es keine Nachkommastellen mehr gibt, sondern nur noch die natürlichen Zahlen von 1 bis 7. 

So kamen der Zusammenhang zwischen Interesse an Mathe und Studienfach und der Zusammenhang zwischen Interesse an Programmieren und Studienfach zustande.

**Comment by reneesch** [comment]
Denkt dran, dass wir nach 23.59 keine Änderungen hier mehr vornehmen dürfen.
